src/RFC-1102_TappletsRegistry.md

```diff
@@ -0,0 +1,263 @@
+DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT HOLDER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL,
+SPECIAL, EXEMPLARY OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR
+SERVICES; LOSS OF USE, DATA OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY,
+WHETHER IN CONTRACT, STRICT LIABILITY OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF
+THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+
+## Language
+
+The keywords "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "SHOULD NOT", "RECOMMENDED",
+"NOT RECOMMENDED", "MAY" and "OPTIONAL" in this document are to be interpreted as described in
+[BCP 14](https://tools.ietf.org/html/bcp14) (covering RFC2119 and RFC8174) when, and only when, they appear in all capitals, as
+shown here.
+
+## Disclaimer
+
+This document and its content are intended for information purposes only and may be subject to change or update
+without notice.
+
+This document may include preliminary concepts that may or may not be in the process of being developed by the Tari
+community. The release of this document is intended solely for review and discussion by the community regarding the
+technological merits of the potential system outlined herein.
+
+## Goals
+
+The aim of this Request for Comment (RFC) is to describe how the Tapplets Registry works and what kind of data about verified and listed Tapplets should be included in it.
+
+## Related Requests for Comment
+
+- [RFC-1100](https://github.com/tari-project/rfcs/pull/134)
+- [RFC-1101](https://github.com/tari-project/rfcs/pull/137)
+
+## Description
+
+The Tapplets Registry is the fundamental part of the Tari Universe described in [RFC-1100](https://github.com/tari-project/rfcs/pull/134). This RFC propose the registry structure as well as manifest file as list of registered and verified Tapplets.
+
+In this document three options are considered:
+
+- Github repository
+- Tari Blockchain - contract to store data onchain
+- Npm package registry (based on MetaMask Snaps)
+
+In next sections different scenarios are discussed as usage examples. Pros and cons of each solution can be found and summarized.
+
+## Github Repository
+
+Github repository is the solution which assumes that every tapplet's metadata is stored in a separate folder, each version in a subfolder. Tapplets themself are distributed az zip bundles, so the code is not stored in the folder. Tapplet Registry manifest file, named `tapplets-registry.manifest.json`, keeps metadata about verified and listed tapplets.
+
+```
+tapplets/
+|- example-tapplet/
+|- example-othet-tapplet/
+|  |- v2.0.0
+|     ├─ dist/
+|     │  ├─ logo.svg
+|     │  ├─ background.svg
+|     ├─ README.md
+|     ├─ tapplet.manifest.json
+|  |- v1.0.1
+|  |- v1.0.0
+tapplets-registry.manifest.json
+```
+
+## Tari Blockchain
```

Comment:
nit: Usually "Tari Blockchain" refers to the Layer 1, since the L2 is not a blockchain `per sê`. Tari Network, or L2, is more unambiguous.